scripts/auto_sshfs_mount.py

```diff
@@ -41,10 +41,11 @@ def AttemptUnmount(local):
             pass
         
 #==============================================================================
-#==============================================================================
 
-if __name__ == '__main__':
+def MountSSHFS():
```

Comment:
The three functions (`mount_sshfs()`, `mount()` and `attempt_unmount`) should be moved to a file in the `huntsman/utils`.  Files in `scripts` should only contain the scripty bits, i.e. import statements, the `if __name__ == '__main__'`, perhaps some argument parsing, then a function call.